frontend-base.json

```diff
@@ -1,18 +1,15 @@
 {
     "extends": [
+        "github>doist/renovate-config:doist-base",
         "config:js-app",
-        "group:linters",
-        "group:monorepos",
         "group:postcss",
-        "group:recommended",
         "group:storybookMonorepo",
         ":automergeMinor",
         ":automergePr",
         ":automergeRequireAllStatusChecks",
         ":disablePeerDependencies",
         ":enableVulnerabilityAlertsWithLabel(security)",
-        ":label(dependencies)",
-        "schedule:weekdays"
+        ":label(dependencies)"
     ],
     "packageRules": [
         {
```

Comment:
Can't comment on the line, but please remove `stabilityDays` from the frontend config as it's part of doist-base 👍